crossbeam-skiplist/README.md

```diff
@@ -35,7 +35,7 @@ extern crate crossbeam_skiplist;
 
 ## Compatibility
 
-The minimum supported Rust version is 1.28. Any change to this is considered a breaking change.
+Crossbeam Skiplist supports stable Rust releases going back at least six months, and every time the minimum supported Rust version is increased, a new minor version is released. Currently, the oldest supported version is 1.36.
```

Comment:
I prefer to use "minimum supported Rust version" instead of "oldest supported version", as the former is a more widely used term.

Reply:
Done